### PR TITLE
improve error message for 404 errors whilst querying Terraform Cloud …

### PR DESCRIPTION
**Pull request by ansgarm**
…workspace

Before:
Screenshot 2021-04-12 at 15 28 54

After:
Screenshot 2021-04-12 at 15 28 18

### CODE REVIEW

**Comment by ansgarm** [comment]
@skorfmann in #488 you mentioned also handling other possible errors. Do you have some example of how to reproduce some?
The response seems to contain some well formed error object which could be printed if any 4xx or 5xx error happens.
Unfortunately it does not differentiate whether the organizationName or workspace caused the 404 so the wording of the error message states both at possibly incorrect.

**Review by skorfmann — APPROVED** [review]
Looks good - let's keep it simple and get it merged 👍

**Comment by ansgarm** [comment]
Example for printing the whole error object because its structure differs between endpoints / methods:
Screenshot 2021-04-12 at 18 11 37

File: packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts [hunk 1]
```diff
@@ -64,7 +64,30 @@ const wait = (ms = 1000) => {
   });
 }
 
-export class TerraformCloud implements Terraform  {
+function BeaufifyErrors(target: any, propertyKey: string, descriptor: PropertyDescriptor) {
```
Comment (skorfmann) [inline packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts:67]: This should probably be `BeautifyErrors`

Comment (ansgarm) [reply packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts]: 🙈

**Comment by skorfmann** [comment]
Cool - let's ship it 👍

**Pull request merged**